index.html

```diff
@@ -101,7 +96,252 @@ <h1>I'm Adeyemi Glad to see you!</h1>
                 </li>  
               </ul>            
             </div>
-            <div> 
+          </div>
+        </section>
+        <Section class="Works">
+          <div class="cardWork">
+            <img id="snap_portfolio">
+            <div class="leftBlock">
+              <div class="pryText_head">
+                <div class="text_head">
+                  Tonic
+                </div>
+              <div class="frame_card">
+                <div class="Client_card">
+                  <div class="Client_txt">
+                    CANOPY
+                  </div>
+                </div>
+                <span class="Counter"></span>
+                <div class="Role_card">
+                  <div class="Role_txt">
+                    Back End Dev
+                  </div>
+                </div>
+                <span class="Counter2"></span>
+                <div class="Year_card">
+                  <p class="Year_txt">
+                    2015
+                  </p>
+                </div>
+              </div>
+            </div>
+            <div class="pryText_body">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+              </p>
+            </div>
+            <ul class="Tags">
+              <div class="textTag_Hml">
+                <li >
+                  <div class="Hml">
+                    <p>html</p>
+                    </div>
+                </li>
+              </div>
+              <div class="textTagcss">
+                <li>
+                  <div class="Css">
+                    <p>css</p>
+                  </div>
+                </li>
+              </div>
+              <div class="textTagjava">
+                <li>
+                  <div class="java">
+                    <p>javascript</p>
+                    </div>
+                </li>
+              </div>
+            </ul>
+              <div id="Action_bttn">
+                <div class="projbttn">
+                  <div class="click_bttn">
+                  <a href="#SeeProject" >See Project</a>
+                </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="cardWork_card_2">
+            <img id="snap_portfolio_2"/>
+            <div class="leftBlock_2">
+              <div class="pryText_head_2">
+                <p class="text_head_2">
+                  Multi-Post Stories
```

Comment:
Kindly put this in `h` tag. This is important in SEO search and also to draw in the reader and give a clear indication of the content on the page ☺